nextcloudappstore/core/models.py

```diff
@@ -123,6 +125,13 @@ def _latest_non_nightly(self, releases):
             return None
 
 
+class AppAuthor(Model):
+    name = CharField(max_length=128, verbose_name=_('Full name'))
+    homepage = URLField(max_length=256, blank=True,
+                        verbose_name=_('Homepage'))
+    mail = EmailField(max_length=128, verbose_name=_('E-Mail'), blank=True)
```

Comment:
Totally nitpicking, but I'd call this field `email`. Mainly because it is the name of the email field on the User model, but also because `mail` makes me think of traditional mail :)